zkvm/src/ops.rs

```diff
@@ -322,18 +322,16 @@ pub enum Instruction {
     /// Immediate data `m` and `n` are encoded as two _LE32_s.
     Cloak(usize, usize),
 
-    /// _value qty_ **fee** → ø
+    /// _qty_ **fee** → _widevalue_
     ///
-    /// 1. Pops an 8-byte _string_ `qty` from the stack and decodes it as _LE64_ integer.
-    /// 2. Pops _value_ from the stack.
-    /// 3. Checks that value.qty is unblinded commitment to `qty` and value.flv is unblinded commitment to zero.
-    /// 4. Adds a _fee entry_ to the _transaction log_.
-    ///
-    /// The checks are performed with group elements directly, without any changes to a _constraint system_.
+    /// 1. Pops an 4-byte _string_ `qty` from the stack and decodes it as _LE32_ integer.
+    /// 2. Checks that `qty` is less or equal to `2^24`.
```

Comment:
2. Checks that `qty` and accumulated fees are less or equal to `2^24`.

Reply:
👍